src/postprocess/chi0.jl

```diff
@@ -181,23 +176,23 @@ function sternheimer_solver(Hk, ψk, ψnk, εnk, rhs;
     bb = R(b -  H(ψk_extra * (ψk_exHψk_ex \ ψk_extra'b)))
     function RAR(ϕ)
         Rϕ = R(ϕ)
-        ARϕ = Rϕ - ψk_extra * (ψk_exHψk_ex \ ψk_extra'H(Rϕ))
-        R(H(ARϕ))
+        Rϕ .-= ψk_extra * (ψk_exHψk_ex \ ψk_extra'H(Rϕ))
+        R(H(Rϕ))
```

Comment:
Isn't this a little misleading? `Rϕ = R(ϕ)` feels right, but with the in-place update that is no longer true. I thought by `A` you mean the Schur complement because then `ARϕ` not a bad variable name, but then you should explain what `A` is.